### ISSUE TITLE
[WARN]gitlab version outdated

### ISSUE THREAD

**Issue by sysdl132** [dwrensha/gitlab-sandstorm#32]
You are using Gitlab 8.7.0 now.But version 12.10 is availble!

**Comment by ocdtrekkie** [comment]
Yeah, unfortunately nobody is currently maintaining Sandstorm's GitLab package. If someone wanted to tackle it, I could help with any questions with packaging and publishing.

**Comment by sysdl132** [comment]
Automatic closed - I am now working on my own website ...
